etc/infrastructure/php/php.ini-development

```diff
@@ -0,0 +1,1932 @@
+[PHP]
```

Comment:
we should avoid specifying the whole `php.ini` file including all the parameters. Instead, we should only specify the custom values we're setting which differs from the default values.

This way we would be more explicit about what are the specific needs of our application without adding all the noise of the default values, and the configuration would be easier to read (just a few lines of code) 🙂

Reply:
At the moment isn't used in the docker-compose file, so maybe it's better to delete it?